src/please.go

```diff
@@ -925,6 +928,14 @@ func (overrides ConfigOverrides) Complete(match string) []flags.Completion {
 	return core.DefaultConfiguration().Completions(match)
 }
 
+// Get an absolute path from a relative path.
+func getAbsolutePath(path string, here string) string {
+	if path[0] != '/' && path[0] != '~' {
+		return here + "/" + path
```

Comment:
I feel that we should we should leverage `filepath.IsAbs` and `filepath.Join` here when handling paths. They will handle relative paths like `../../this/dir` for you, among other things, such as providing cross-compatibility between different OSes.